src/index.css

```diff
@@ -0,0 +1,68 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
- Kindly work again on your wireframe: The App must be properly styled and its layout looks like the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png)
- Your design.

![Capture](https://user-images.githubusercontent.com/63972670/202489958-be86df9c-7f63-4a4c-8efb-b678a5bb93d1.PNG)
- What the requirements say:
- 
![Capture](https://user-images.githubusercontent.com/63972670/202490472-4cc414a7-eaa7-436a-b1c5-66719c5ee31a.PNG)

Reply:
thanks for suggesting me , I accidently quest the day 3 project on the day 4 submissions and this mistake happen. for styling i have not even started. But when i try to do that, i will take your suggestion to my heart and try my best. Thank you ✨